infrastructure/ansible/roles/ats/defaults/main.yml

```diff
@@ -14,7 +14,16 @@
 #
 
 install_ats: False
-ort_pkg_name: "{% if ort_version is defined and ort_version != omit %}{{ 'traffic_ops_ort-'+ort_version }}{% else %}traffic_ops_ort{% endif %}"
+ort_pkg_name: >
```

Comment:
Since you don't want extra whitespace or newlines, you should use `>-` and the jinja interpolation tokens need `{%-` and `-%}` appropriately.

Reply:
I thought `: >` would collapse newlines and extra spaces ?

Reply:
Ok, fixed